.github/workflows/provider.yml

```diff
@@ -0,0 +1,95 @@
+name: Provider
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php-version }} / ${{ matrix.provider }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.3', '7.4', '8.0']
+        provider:
```

Comment:
We could do something like this if we don't want to list all providers:

https://github.com/brefphp/extra-php-extensions/blob/master/.github/workflows/ci.yml#L19

Reply:
I must admit I'm not sure what the `echo "::set-output name=list::$(ls layers | jq -R -s -c 'split("\n")[:-1]')"` command does 😁